igcollect/logfile_values.py

```diff
@@ -125,19 +125,33 @@ def get_speed(self):
         # Speed :-/?
         return self.get_sum() / self.get_timeshift()
 
+    def get_distribution(self):
+        d = {}
+        uniq_values = set(self.values)
```

Comment:
Do I get it right, that the list `self.values` doesn't hold complex types like other `list`s oder `dict`s?

Reply:
self.values are numbers